packages/amplify-codegen-e2e-tests/src/__tests__/push-codegen-admin-modelgen.test.ts

```diff
@@ -0,0 +1,19 @@
+import { DEFAULT_JS_CONFIG, createNewProjectDir } from "@aws-amplify/amplify-codegen-e2e-core";
+import { deleteAmplifyProject, testPushCodegen } from "../codegen-tests-base";
+
+const schema = 'admin-modelgen.graphql';
+
+describe('Amplify push with codegen tests - admin modelgen', () => {
```

Comment:
Here and throughout this file and the associated schema: I don't see where admin enters into this--am I missing some setup from elsewhere?

Reply:
I've updated the specific steps in `testPushAdminModelgen`